justfile

```diff
@@ -19,20 +18,20 @@ clean:
   rm -f compile_commands.json
 
 build *args="//...":
-  bazel build {{args}}
+  bazel build {{args}} --test_env=LLVM_SYMBOLIZER=llvm-symbolizer
```

Comment:
Why not just edit bazel so that this is set as the default value if the environment variable isn't set?

Reply:
I agree, let's add this to .bazelrc

Reply:
I see that we already have `build:unix --test_env=LLVM_SYMBOLIZER=llvm-symbolizer` inside `.bazelrc` file, and it doesn't work for me running on unix. I still need to pass `--test_env=LLVM_SYMBOLIZER=llvm-symbolizer`

Reply:
wat? If it is indeed linux or macos, then there's

```
build:linux --config=unix
build:macos --config=unix
```

somethis is broken in your config?